crypto/hpke/hpke.c

```diff
@@ -263,138 +240,105 @@ static int hpke_aead_dec(OSSL_LIB_CTX *libctx, const char *propq,
  * @param ivlen is the length of the iv
  * @param aad is the additional authenticated data
  * @param aadlen is the length of the aad
- * @param plain is an output
- * @param plainlen is the length of plain
- * @param cipher is an output
- * @param cipherlen input/output, better be big enough on input, exact on output
- * @return 1 for good otherwise bad
+ * @param pt is the plaintext buffer
+ * @param ptlen is the length of pt
+ * @param ct is a caller-allocated output buffer
```

Comment:
I know what you mean here, but it could be on the stack.. Just the output buffer is sufficient

Reply:
ack